java/drivers/driver-hazelcast4plus/src/main/java/com/hazelcast/simulator/tests/map/prunability/KeyPojo.java

```diff
@@ -0,0 +1,38 @@
+package com.hazelcast.simulator.tests.map.prunability;
+
+import com.hazelcast.partition.PartitionAware;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+public class KeyPojo implements Serializable, PartitionAware<Integer> {
```

Comment:
KeyPojo does not implement IDS - AFAIR in perf tests we usually implement it, so we do not get additional overhead of java serialization

Reply:
The tests have the same conditions, is serialization a problem, in your opinion?

Reply:
Okay, I got your point. 
The reason why I took java ser/de -- it's just the most commonly used ser/de. If we want to change ser/de - I am not in favor of using IDS. I'd pick compact ser/de, since we consider it as a default one.